### ISSUE TITLE
sow.pfron.org.pl

### ISSUE THREAD

**Issue by Lukjan38** [MajkiIT/polish-ads-filter#19368]
mimo akceptacji cookies, nadal sie pojawia

**Comment by krystian3w** [comment]
Jak akceptacja nie działa to pisz do webmastera strony by naprawił ją.

**Comment by krystian3w** [comment]
Dodatkowo nie mam konta dla niepełnosprawnych (rodzinie lekkie upośledzenie umysłowe powygasało).

Więc umów się z @hawkeye116477 na zdalne połączenie z użyciem TeamViewer / AnyDesk.

**Comment by felix-22** [comment]
A u mnie po akceptacji, już się nie pojawia.

**Comment by krystian3w** [comment]
Ja mam wycięte, naprawcie przeglądarki jak nie działa filtr:

`##.cookie-info-overlay` od Kiboke: https://majkiit.github.io/polish-ads-filter/#i_dont_care_about_cookies

Jak po zalogowaniu jest inny pop-up to ja nie mam szans go wyciąć.

**Comment by felix-22** [comment]
Hmm, no u mnie też wycięte, Lukjan38 co ty w kulki lecisz...po prostu ja sprawdzałem na drugim profilu, na którym nie mam list ciasteczkowych, żeby zobaczyć czy zniknie po akceptacji, bo taki zgłaszał problem, a nawet nie przypuszczałem, że tu ta kie kwatki wyjdą, że zgłoszenie jest fałszywe...domniemałem, że nie jest...

**Comment by krystian3w** [comment]
Podobno to może być od `Komponowanie:	WebRender`

**Comment by felix-22** [comment]
> wywala a nie tylko zawiesza.

No to samo miałem na myśli, jak się zawiesi to już się nie odwiesi i trzeba proces zabijać, może to efekt zwolnienia 250 pracowników Mozilli pod koniec 2020 roku i teraz efektem tego jest Firefox kulejący na jakości...
`https://www.google.pl/search?q=mozilla+zwalnia+250+pracownikow&source=hp`

**Comment by Lukjan38** [comment]
> Dodatkowo nie mam konta dla niepełnosprawnych (rodzinie lekkie upośledzenie umysłowe powygasało):
> 
> > Użytkownik (w Profilu Zaufanym: `***@***.pl`), którym próbujesz się zalogować do systemu SOW **nie istnieje**.
> 
> Więc umów się z @hawkeye116477 na zdalne połączenie z użyciem TeamViewer / AnyDesk.

musisz sie najpierw zarejestrowac na stronie(z takimi danymi jak w profilu zaufanym), potem zalogowac przez profil zaufany.

**Comment by krystian3w** [comment]
Ale ja mam tylko dysleksję (błędy ortograficzne/literówki/niedostrzeganie różnic między `###` a `####`), więc pewnie odrzucą ręczną rejestracje online.

**Comment by Lukjan38** [comment]
aha. 

czyli tego nie idzie naprawic?

**Comment by krystian3w** [comment]
Musisz z kolejny raz sprawdzić w uBO czy masz zasubskrybowane filtry I don't care about cookies:

https://majkiit.github.io/polish-ads-filter/#i_dont_care_about_cookies

Wtedy pop-up dolny powinien być ukrywany, jak będzie migać to raczej PC wymaga modernizacji i trzeba dać odejść staremu AMD "atlon" procesorom Intel Xeon ("serwerowym") i Core 2 Duo mimo, że SSD przypśpieszyło PC.

**Comment by Lukjan38** [comment]
mam dodanie te filtry.

ja mam amd ryzen 5 3400g.

**Comment by krystian3w** [comment]
Mnie on znika z tamtą listą, jak dezaktywuje filtry kosmetyczne to też pop-up się zamyka - czyli jak czujesz mus kliknięcia "akceptuj" to powinno działać i nie wracać (jak używasz Auto Delete Cookie to będzie nawracać mimo klikania).

Jak masz aktywne `EasyList Cookie` lub Fanboy **Annoyance** to ja bym je wyłączył..., że MajkiIT tego używa jako zamiennik to nie znaczy, że wyłapuje błędne filtry i monitoruje stan napraw. Dodatkowo pewnie fanboje nie posiadają generycznego filtru "cookie-info-overlay".

Możliwe, że też jakimś cudem np. https://github.com/bogachenko/fuckfuckadblock/ dodał generichide dla tej strony, stąd komunikat wycieka - jak jego lista psuje to ją wyłącz lub próbuj zgłosić jemu błąd by zmniejszył użycie generichide jak już nie ma potrzeby dla strony "rządowej".

**Comment by Lukjan38** [comment]
aha, czyli musze miec to i to?

**Comment by krystian3w** [comment]
Nie, tamte 3 listy ja zalecam powyłączać.

Zagraniczne komunikaty adblock zgłaszaj tu: https://github.com/uBlockOrigin/uAssets/issues

**Comment by Lukjan38** [comment]
daj linki co dodac i wylaczyc, abym sie nie pomylil :)

**Comment by krystian3w** [comment]
Znowu dezaktywowałeś I don't care about cookies, ono musi zostać włączone:
https://majkiit.github.io/polish-ads-filter/#i_dont_care_about_cookies

**Comment by Lukjan38** [comment]
ok, a reszta mam dobrze?

**Comment by krystian3w** [comment]
Tak, reszta nie powinna robić złej interferencji, że pop-up wycieka.

**Comment by Lukjan38** [comment]
ok, dzieki, jest ok.

czy jest skuteczny, filtr na wp i podstroony?

**Comment by felix-22** [comment]
Biedny Lukjan38: https://github.com/MajkiIT/polish-ads-filter/issues/17877#issuecomment-808863992

> Lukjan38 : czy jest skuteczny, filtr na wp i podstrony?

Skutecznego nie ma, każdy pada po jakimś czasie, zazwyczaj kilku/kilkunastu dniach, niestety nikomu, poza AdGuard, nie chce się ciągle naprawiać WP, właśnie z tego powodu gzenio22 przerzucił się na AdGuard.

**Comment by krystian3w** [comment]
Jak felix będzie co tydzień podsyłać nowy na podstawie analizy lub podkradania pomysłów z AdGuard Base.

Niby nie zawsze aktywacja AdGuard Base równa się, że ich skryptlet naprawi WP z uBlock Origin.

**Comment by Lukjan38** [comment]
aha, szkoda :(

**Comment by felix-22** [comment]
> Jak felix będzie co tydzień podsyłać nowy na podstawie analizy lub podkradania pomysłów z AdGuard Base.

Chciało mi się do czasu, po miesiącu/dwóch przestało mi się chcieć, i nie tylko mi się odechciało, bo ludziom od list Adblock Plus i uBlock Origin też, zgłoszenia pozamykane, a reklamy dalej hulają:
- https://github.com/abp-filters/abp-filters-anti-cv/issues/505#issuecomment-756717531
- https://github.com/uBlockOrigin/uAssets/issues/8261


> wizmak : https://github.com/abp-filters/abp-filters-anti-cv/issues/505#issuecomment-756717531 :
> ANGIELSKI:
> As you already know these are CV ads of which some are already blocked at the moment. @adblock-filters is submitting fixes whenever possible, but unfortunately there is not a permanent long-lasting solution that blocks 100% of these ads.
> 
> I'll close this issue if you don't mind, and thank you for reporting!
>
> POLSKI:
> Jak już wiesz, są to reklamy CV, z których niektóre są w tej chwili zablokowane. @ adblock-filters przesyła poprawki, gdy tylko jest to możliwe, ale niestety nie ma trwałego rozwiązania, które blokowałoby 100% tych reklam.
> 
> Zamknę ten problem, jeśli nie masz nic przeciwko, i dziękuję za zgłoszenie!

**Comment by felix-22** [comment]
[Poszło](https://github.com/MajkiIT/polish-ads-filter/issues/17375#issuecomment-809677900) na Listę Fałszywek.

**Comment by Lukjan38** [comment]
czemu?

**Comment by krystian3w** [comment]
Bo komunikat da się wyciąć (wycina lista IDCAC), akceptacja działa - więc nie psuje jej filtr sieciowy.

Nie widzę by hawkeye sprawdzał co u Ciebie się dzieje po zalogowaniu, że komunikat nie znika lub nie działa przycisk akceptuj.

---

Ja by raczej odchodził z kopiowania filtrów od kiboke (IDCAC) skoro nas wyprzedza.

**Comment by Lukjan38** [comment]
aha, juz nie mam, musialem dodac filtry z I don't care about cookies
